app/lib/http/request.ts

```diff
@@ -0,0 +1,24 @@
+const BASE_END_POINT = "https://api.themoviedb.org/3";
+
+const DEFAULT_HEADERS = {
+  "Content-Type": "application/json",
+  Accept: "application/json",
+};
+
+export type GetMethodParams = {
+  path: string;
+  headers?: { [key: string]: string };
+  params?: { [key: string]: string };
+};
+
+export const get = async ({ path, headers = DEFAULT_HEADERS, params }: GetMethodParams) => {
+  const endpoint = `${BASE_END_POINT}/${path}?api_key=${process.env.NEXT_PUBLIC_API_KEY}`;
+  const queryString = params && new URLSearchParams(params).toString();
+
+  const request = new Request(queryString ? `${endpoint}&${queryString}` : endpoint, {
+    method: "GET",
+    headers,
+  });
+
+  return fetch(request);
```

Comment:
You might want to set `revalidate` depending on the request.

> By default, Next.js automatically caches the returned values of fetch in the [Data Cache](https://nextjs.org/docs/app/building-your-application/caching#data-cache) on the server. This means that the data can be fetched at build time or request time, cached, and reused on each data request.

ref: https://nextjs.org/docs/app/building-your-application/data-fetching/fetching-caching-and-revalidating#revalidating-data

Reply:
I disabled it by default, thank you for catching it!